src/explore-education-statistics-admin/src/modules/find-statistics/PublicationReleaseContent.tsx

```diff
@@ -51,33 +48,45 @@ const PublicationReleaseContent = () => {
           body: '',
         },
       });
-  }, [release, addContentSectionBlock]);
+  }, [release, actions]);
```

Comment:
As `actions` is just an object (and isn't memoized), this will cause this callback to inefficiently change on every render. We need to either:

- Memoize `actions` using `useMemo`
- Define the `useCallback` deps as `actions.addContentBlock` rather than `actions`.

Same with all the other `useCallback` deps in this component (and their relevant action).